### PR TITLE
Increase the size of the decoder buffer to 1meg.

### PR DESCRIPTION
**Pull request by mikeal**
The current default is far too small, it's breaking on a node I have that is only 66015 bytes.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -30,7 +30,8 @@ const decoder = new cbor.Decoder({
       val = val.slice(1)
       return {'/': val}
     }
-  }
+  },
+  size: 1e+6 /* one megabyte */
```
Comment (vmx) [inline src/util.js]: Looking at the implementation, it tries to use power of two. So it make make sense to write is as `size: 1024 * 1024 * 1024` (which I find easier to read, but that's a matter of taste).

**Comment by vmx** [comment]
@mikeal I guess you still want this in?

**Comment by mikeal** [comment]
@vmx the default should **definitely** be larger, but it's not clear to me how much larger. Because of the way CBOR is implemented it requires a static allocation of memory of double the max node size so there will be tradeoffs on increasing it as well.

**Comment by vmx** [comment]
Should we make the default to a value that Bitswap can still transfer, hence 4MB (see https://github.com/ipld/js-ipld-dag-cbor/issues/52 for more discussions on that topic).

**Pull request closed without merging**